LSPS1/README.md

```diff
@@ -283,19 +284,78 @@ The client MAY check the current status of the order at any point.
 
 ### 3. Payment
 
-This section describes the `payment` object returned by `lsps1.create_order` and `lsps1.get_order`. The client MUST pay the `bolt11_invoice` OR the `onchain_address`. Using both methods MAY lead to the loss of funds.
+This section describes the `payment` object returned by `lsps1.create_order` and `lsps1.get_order`.
 
-> **Rationale** On-chain payments are required for payments with higher amounts, especially to push `client_balance_sat` to the client. On-chain payments are also useful to onboard new users to Lightining. On the other hand, Lightning payments are the preferred way to do payments because they are quick and easily refundable.
+```json
+"payment": {
+    "onchain": {...}.
+    "bolt11": {...}
+}
+```
+
+An LSP MAY omit payment options.
+> **Rationale**: E.g: The LSP might not support onchain payments.
+
+> **Rationale**:
+> Fees and expiry dates are defined on the level of the payment-option.
+> This allows an LSP to provide the cheapest service for each payment option.
+>
+> An LSP might choose to have a lower fee for lightning than for onchain payments.
+> Onchain payments might
+> - be more costly for the LSP as they might have to spend a small UTXO in the future
+> - be more risky for the LSP in case of 0-conf channels
+> - require longer confirmation times than a lightning payment. (Risk of changing on-chain fees)
 
 
-**Payment object**
+#### 3.1 Lightning Payments using BOLT-11
 
 ```json
-"payment": {
+"bolt11": {
+    "state" : "EXPECT_PAYMENT",
+    "bolt11_invoice": "lnbc252u1p3aht9ysp580g4633gd2x9lc5al0wd8wx0mpn97..."
+    "fee_total_sat": 8888,
+    "order_total_sat": "200888",
+    "state": "EXPECT_PAYMENT"
+}
+```
+
+- `state`
+    - `EXPECT_PAYMENT` Payment expected.
+    - `HOLD` Lighting payment arrived, preimage NOT released.
+    - `PAID`  When the has been reimage released
+    - `REFUNDED` Lightning payment has been refunded.
+- `expires_at` <[LSPS0.datetime][]> The timestamp at which the
+- `fee_total_sat` <[LSPS0.sat][]> The total fee the LSP will charge to open this channel in satoshi.
+- `order_total_sat` <[LSPS0.sat][]> What the client needs to pay in total to open the requested channel.
+    - MUST be the `fee_total_sat` plus the `client_balance_sat` requested in satoshi.
+- `bolt11_invoice <string>`
+    - MUST be a [Lightning BOLT 11 invoice](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) for the number of `order_total_sat`. 
+    - Invoice MUST be a [HOLD invoice](https://bitcoinops.org/en/topics/hold-invoices/).
+    - MUST be at most 2048 characters long.
+
+**Client**
+
+- MAY pay the `bolt11_invoice`.
+- MAY pull `lsps1.get_order` to check the success of the payment.
```

Comment:
Do we need a notification here `lsps1.bolt11_holding`?

Reply:
I think we can build the UX we need without `lsps1.bolt11_holding`-notifications.